lib/src/builder/field_header_widget.dart

```diff
@@ -13,13 +12,13 @@ class FieldHeader extends StatelessWidget {
       children: [
         Text(
           '${property.title} ${property.required ? "*" : ""}',
-          style: WidgetBuilderInherited.of(context).uiConfig.fieldTitle,
+          style: Theme.of(context).textTheme.bodySmall,
         ),
         if (description != null && description.isNotEmpty) ...[
           const SizedBox(height: 4),
           Text(
             description,
-            style: WidgetBuilderInherited.of(context).uiConfig.fieldDescription,
+            style: Theme.of(context).textTheme.bodyMedium,
```

Comment:
It's kinda weird that the title has `bodySmall` while the description (smaller text, or less bold) has `bodyMedium`, maybe something we didn't consider when we were updating our text themes?

Reply:
Yeah, we can also change it here as this won't directly affect the app unless we fetch these changes, then update it in our theming PR, so we can merge this one :)

Reply:
I switched them bodyMedium in https://github.com/doneservices/done-flutter/pull/1589 and also here now.

Reply:
Could we also Cherrypick this commit from https://github.com/doneservices/done-flutter/pull/1589 into a new branch and merge it in main?